src/mame/drivers/pc9821.cpp

```diff
@@ -637,14 +637,16 @@ MACHINE_RESET_MEMBER(pc9821_state,pc9821)
 // TODO: setter for DMAC clock should follow up whatever is the CPU clock
 
 // TODO: remove me, cfr. pc9801.cpp
-#define MAIN_CLOCK_X2 2'457'600
+#define BASE_CLOCK      XTAL(31'948'800)    // verified for PC-9801RS/FA, TODO: verified for 9821 series
```

Comment:
`TODO: verified` sounds like a typo, should be `TODO: verify for PC-9821`?
Also I suggest to merge this TODO with the one just above (IDEs generally expect to read these with a `// TODO: (.*)` pattern.

Reply:
Ah, that's indeed a typo - thanks!  
It should be fixed now.